src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -86,7 +86,7 @@
           text: az storage account create -n MyStorageAccount -g MyResourceGroup -l westus --sku Standard_LRS
           min_profile: latest
         - name: Create a storage account 'MyStorageAccount' in resource group 'MyResourceGroup' in the West US region with locally redundant storage.
-          text: az storage account create -n MyStorageAccount -g MyResourceGroup -l westus --account-type Standard_LRS
+          text: az storage account create -n MyStorageAccount -g MyResourceGroup -l westus --sku Standard_LRS
```

Comment:
I believe this is actually incorrect. Note `max_profile: 2017-03-09-profile`. In the old storage create command, this was actually called `--account-type`. Does the rule check for profile dependency?